### PR TITLE
installing the loki operator using OCP-nerc-configs methodology

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
installing the loki operator using knowledge transfer from NERC

Signed-off-by: greg pereira [grpereir@redhat.com](mailto:grpereir@redhat.com)

addresses part 1 of: https://github.com/operate-first/apps/issues/2493
corressponds to: https://github.com/OCP-on-NERC/nerc-ocp-config/pull/69/files

This PR was formatted this way because it attempts to keep as much of the existing structure from the OCP-on-NERC gitops repo. There was some deviation from the orrigional pr however, as soon after the operatorgroup and subscription were abstracted out into cluster-scope. Everything else lives in `loki/base`.

/cc @HumairAK

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
bump @HumairAK 
/cc @durandom

**Comment by HumairAK** [comment]
Have we checked with NERC folks to see why cluster [rbac](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/loki/base/clusterrolebindings) is in the `loki` folder and not their [cluster-scope](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/cluster-scope/base/rbac.authorization.k8s.io) folder? This seems anti-pattern for the current set up (for both operate-first and nerc) 

@larsks any thoughts?

**Comment by Gregory-Pereira** [comment]
> Have we checked with NERC folks to see why cluster [rbac](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/loki/base/clusterrolebindings) is in the `loki` folder and not their [cluster-scope](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/cluster-scope/base/rbac.authorization.k8s.io) folder? This seems anti-pattern for the current set up (for both operate-first and nerc)

Thanks for the reminder I forgot about this. Hopefully we can get some input from Lars here, but if he doesn't happen to see it, I have also [put this question to the NERC team](https://massopencloud.slack.com/archives/C027TDE52TZ/p1666623728336859) in the MOC slack.

File: grafana/overlays/emea/jerry/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+kind: Kustomization
+namespace: opf-monitoring
+generatorOptions:
+  disableNameSuffixHash: true
+configMapGenerator:
+  - name: grafana-config-overrides
+    literals:
+      - GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_STRICT=true
+resources:
+  - ../../../base
```
Comment (durandom) [inline grafana/overlays/emea/jerry/kustomization.yaml:12]: dont you need to include this kustomization in the jerry deployments?
And I don't know if we have the grafana operator deployed on jerry

**Review by durandom — COMMENTED** [review]
looks overall correct to me. see the grafana comment

File: cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/loki/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - lokistack-gateway-auth-delegator.yaml
+  - lokistack-tenant-logs.yaml
```
Comment (tumido) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/loki/kustomization.yaml:5]: Please use the pattern we always use and don't create new ones. Read this readme please:

https://github.com/operate-first/apps/tree/master/cluster-scope#application-folder-structure

Each resource has it's own folder. If you need to bundle resources together, you can create new bundle in `clsuter-scope/bundles`.

**Review by tumido — CHANGES_REQUESTED** [review]

**Pull request closed without merging**